Cargo.lock

```diff
@@ -12392,6 +12495,34 @@ dependencies = [
  "tracing-futures",
 ]
 
+[[package]]
+name = "tonic"
+version = "0.9.2"
```

Comment:
maybe we should look at what it would take to update tonic so we don't have duplicate deps here

Reply:
Upgraded us to 0.10, hopefully otel follows suit sometime soon.